sass/README.md

```diff
@@ -0,0 +1,41 @@
+SASS Style Guide
+=====
+### Formatting
+* Use the SASS syntax.
+* Use hyphens when naming mixins, extends, classes & variables: `span-columns` not `span_columns` or `spanColumns`.
+* Use space between property and value: `width: 20px` not `width:20px`.
+* Use a blank line above selector that has styles.
+* Prefer hex color codes `#000`.
+* Use `//` for comment blocks not `/* */`.
+* Use single or double quotes consistently. Preference is for double quotes, e.g., `content: ""`.
+* Use only lowercase, including colors.
+* Don't add a unit specification after `0` values, unless required by a mixin.
+
+### Order
+* Place scoped variables, @extends and @includes (excluding media query stuff) at the top of your declaration list, in that order.
+* Use alphabetical order for declarations.
```

Comment:
We should either enforce the use of this order on our projects, or change this to something else.

I find really "unnice" for each one to just go ahead and use the order of her/his preference. We should, as a team reach a consensus, and document it. 

We had this discussion before, and then the winner was this order, but then we see projects were this is not happening. That should be strongly discouraged, regardless of which is the style we, as a team, decide to go with.